templates/oauth2/patient-select.html.twig

```diff
@@ -0,0 +1,123 @@
+            <div class="text-md-center">
+                <h4 class="mb-4 mt-1">{{ "Patient Selection"|xlt }}</h4>
+            </div>
+            {% if patients|length < 1 %}
+            <div class="row w-100">
+                <div class="col">
+                    {% if errorMessage is not empty %}<p class="alert alert-warning">{{ errorMessage|text }}</p>{% endif %}
+                    <p class="alert alert-info">{{ "No patients to select"|xlt }}</p>
+                </div>
+            </div>
+            {% else %}
+            <div class="row w-100">
+                <div class="col">
+                    <form action="{{ searchAction|attr }}" method="GET">
+                        <input class="w-25" name="search[fname]" type="text" class="form-control form-input" placeholder="{{ "First Name"|xla }}"
+                        value="{{ fname|attr }}" />
+                        <input class="w-25" name="search[mname]" type="text" class="form-control form-input" placeholder="{{ "Middle Name"|xla }}"
+                        value="{{ mname|attr }}" />
+                        <input class="w-25" name="search[lname]" type="text" class="form-control form-input" placeholder="{{ "Last Name"|xla }}"
+                        value="{{ lname|attr }}" />
+                        <input type="submit" value="{{ "Search"|xla }}" />
+                    </form>
+                    {% if hasMore %}
+                    <p class="alert alert-info">{{ "Too many search results found. Displaying a limited set of patients. Narrow your search results through the filters above."|xlt }}</p>
+                    {%  endif %}
+                </div>
+            </div>
+            <div class="row w-100">
+                <p class="col">
+
+                <table class="table table-striped">
+                    <thead>
+                    <tr>
+                        <th>{{ "Name"|xlt  }}</th>
+                        <th>{{ "DOB"|xlt  }}</th>
+                        <th>{{ "Sex"|xlt  }}</th>
+                        <th>{{ "Email"|xlt  }}</th>
+                        <th></th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    {% for patient in patients %}
+                    <tr>
+                        <td>
+                            {{ patient.fname|text }}{% if patient.mname %} {{ patient.mname|text }}{% endif%} {{ patient.lname|text }}
+                        </td>
+                        <td>
+                            {{ patient.DOB|text }}
+                        </td>
+                        <td>
+                            {{ patient.sex|text }}
+                        </td>
+                        <td>
+                            {{ patient.email|text }}
+                        </td>
+                        <td>
+                            <button data-patient-id={{ patient.uuid|attr }}" class="btn btn-primary patient-btn">{{ "Select patient"|xlt  }}</button>
```

Comment:
missing the double quote after `ata-patient-id=`

Reply:
Thanks, fixed.

Reply:
Do we have a class patient-btn defined in our themes or is this custom?

Reply:
It is custom for the javascript to key off of.